epochStart/bootstrap/disabled/disabledStorer.go

```diff
@@ -0,0 +1,87 @@
+package disabled
+
+import (
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"sync"
+)
+
+// Storer -
+type Storer struct {
+	mut  sync.Mutex
+	data map[string][]byte
+}
+
+// NewDisabledStorer -
+func NewDisabledStorer() *Storer {
+	return &Storer{
```

Comment:
we already have an inmemory storer. you could use that.

Reply:
that inmemory storer only implements the persister interface, not the storer